### PR TITLE
API test for sharing files when receiver has matching files

### PR DESCRIPTION
**Pull request by bhawanaprasain**
## Description
It includes API test when receiver has matching file name .

## Related Issue
#35414

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [X] Tests only (no source changes)

### CODE REVIEW

**Comment by bhawanaprasain** [comment]
@individual-it @phil-davis please review on my PR

**Review by phil-davis — APPROVED** [review]

File: tests/acceptance/features/apiShareManagement/acceptShares.feature [hunk 1]
```diff
@@ -430,3 +430,22 @@ Feature: accept/decline shares coming from internal users
...
+    And user "user1" shares file "/testfile.txt" with user "user2" using the sharing API
+    And user "user2" accepts the share "/testfile.txt" offered by user "user1" using the sharing API
+    And user "user2" accepts the share "/testfile.txt" offered by user "user0" using the sharing API
+    Then the sharing API should report to user "user2" that these shares are in the accepted state
+      | path                  |
+      | /testfile (2).txt     |
+      | /testfile (2) (2).txt |
+    And the content of file "/testfile.txt" for user "user2" should be "file from user2"
+    And the content of file "/testfile (2).txt" for user "user2" should be "file from user1"
+    And the content of file "/testfile (2) (2).txt" for user "user2" should be "file from user0"
```
Comment (phil-davis) [inline tests/acceptance/features/apiShareManagement/acceptShares.feature]: That is a bit weird. I wonder why the file is not called `/testfile (3).txt`

Comment (phil-davis) [reply tests/acceptance/features/apiShareManagement/acceptShares.feature]: I raised issue #35459 - others can comment if they think it is a bug or a feature.

**Comment by phil-davis** [comment]
Backport `stable10` #35537

**Pull request merged**